.github/workflows/build-package.yml

```diff
@@ -3,84 +3,79 @@ name: Build Package
 on:
   workflow_call:
     inputs:
-      os:
-        description: The list of operating systems to use in the strategy matrix.
-        type: string
-        required: false
-        default: '["ubuntu18.04", "ubuntu20.04", "debian9"]'
-      variant:
-        description: The list of variants to use in the strategy matrix.
-        type: string
-        required: false
-        default: '[{"arch": "arm", "platform": "linux/arm/v7"}, {"arch": "arm64", "platform": "linux/arm64/v8"}, {"arch": "amd64", "platform": "linux/amd64"}]'
       artifact:
-        description: The artifact to publish.
+        description: The artifact to create containing the packages.
```

Comment:
Confused by this statement, do you mean artifact containing packages? Suggest re-wording

Reply:
Agreed, this is confusing! Updating to make it more clear 👍🏻